.pre-commit-config.yaml

```diff
@@ -26,7 +26,7 @@ repos:
           'flake8-docstrings==1.5.0',  # Verifies that all functions/methods have docstrings
           'flake8-type-annotations==0.1.0',  # Looks for misconfigured type annotations
           'flake8-annotations==1.1.1',  # Enforces type annotation
-          'flake8-logging-format==0.6.0',
+          'flake8-logging-format==0.6.0',  # Enforces log formatting to be correct
```

Comment:
From the `flake8-logging-format` docs it seems that %-formatted loggers are flagged.

Maybe we could ignore this warning to enable use of %-formatted logs?

```python
G002 Logging statements should not use % formatting for their first argument
```

I don't think logging variables using `extra` is a good idea for a python module. This could potentially lead to name collisions that a user of the library couldn't possibly foresee.

Reply:
This is correct.  
From the [docs](https://docs.python.org/3.9/howto/logging.html#logging-variable-data), log messages should be in this syntax:

```py
logging.debug('log message: %s', variable)
```
and not
```py
logging.debug('log message, %s' % variable)
```
The `logging.x` should take input variables, while using `%` instead would interpolate as discussed in #18